tests/unit/test_network_connections.py

```diff
@@ -267,17 +268,21 @@ def do_connections_validate_nm(self, input_connections, **kwargs):
                     s6.clear_routes()
                     for r in kwargs["nm_route_list_current"][idx]:
                         r = parser.validate(r)
-                        r = NM.IPRoute.new(
+                        rr = NM.IPRoute.new(
```

Comment:
I renamed the var from `r` into `rr` for the sake of consistency of the var name in `./library/network_connections.py`,  
https://github.com/linux-system-roles/network/blob/main/library/network_connections.py#L1133

Reply:
Note for the future: We need to translate `r` and `rr` to meaningful names.